packages/esm-patient-chart-app/src/deceased/deceased.resource.ts

```diff
@@ -0,0 +1,35 @@
+import { openmrsFetch, openmrsObservableFetch } from '@openmrs/esm-framework';
+
+export async function setDeceased(payload: any, personUuid: string, abortController: AbortController) {
+  return openmrsFetch(`/ws/rest/v1/person/${personUuid}`, {
+    signal: abortController.signal,
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: payload,
+  });
+}
+
+export async function getConceptAnswers(controller: AbortController) {
+  const conceptUuid = await getCauseOfDeathConcept().then((res) => res.data.value);
```

Comment:
Instead of doing two synchronous network requests wouldn't it be better to have an endpoint which would give concept name based on a human readable key? The first endpoint is asking for `concept.causeOfDeath`. Why is it not supported to request from `/concept/concept.causeOfDeath` ?